schemas/com.snowplowanalytics.snowplow.ecommerce/snowplow_ecommerce_action/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for an Ecommerce action",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "snowplow_ecommerce_action",
```

Comment:
This is really nit-picky so feel free to ignore - the vendor is already `com.snowplowanalytics.snowplow.ecommerce` so the name here feels like it could be just `action`.  The fact that it is from Snowplow and related to ecommerce is already stated in the vendor.  Wdyr?

Reply:
It was explicitly set like that to assist in the modeling way :)